src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/SolutionExplorerInProcess.cs

```diff
@@ -207,14 +207,28 @@ public async Task<string[]> GetProjectReferencesAsync(string projectName, Cancel
             return references;
         }
 
-        public async Task AddProjectAsync(string projectName, string projectTemplate, string languageName, CancellationToken cancellationToken)
+        public Task AddProjectAsync(string projectName, string projectTemplate, string languageName, CancellationToken cancellationToken)
+            => AddProjectAsync(projectName, projectTemplate, projectTemplateAdditionalParameters: string.Empty, languageName, cancellationToken);
+
+        public async Task AddProjectAsync(string projectName, string projectTemplate, string projectTemplateAdditionalParameters, string languageName, CancellationToken cancellationToken)
         {
             await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 
             var projectPath = Path.Combine(await GetDirectoryNameAsync(cancellationToken), projectName);
             var projectTemplatePath = await GetProjectTemplatePathAsync(projectTemplate, ConvertLanguageName(languageName), cancellationToken);
-            var solution = await GetRequiredGlobalServiceAsync<SVsSolution, IVsSolution6>(cancellationToken);
-            ErrorHandler.ThrowOnFailure(solution.AddNewProjectFromTemplate(projectTemplatePath, null, null, projectPath, projectName, null, out _));
+
+            if (string.IsNullOrEmpty(projectTemplateAdditionalParameters))
+            {
+                var solution = await GetRequiredGlobalServiceAsync<SVsSolution, IVsSolution6>(cancellationToken);
+                ErrorHandler.ThrowOnFailure(solution.AddNewProjectFromTemplate(projectTemplatePath, null, null, projectPath, projectName, null, out _));
+            }
+            else
```

Comment:
I add this else block as a trial and error because the existing `AddNewProjectFromTemplate` wasn't able to handle the wizard. You may want to see if we can avoid this if-else and instead just have the following code:

```csharp
            if (!string.IsNullOrEmpty(projectTemplateAdditionalParameters))
                projectTemplatePath += "|" + projectTemplateAdditionalParameters;
            var solution = await GetRequiredGlobalServiceAsync<SVsSolution, IVsSolution6>(cancellationToken);
            ErrorHandler.ThrowOnFailure(solution.AddNewProjectFromTemplate(projectTemplatePath, null, null, projectPath, projectName, null, out _));
```